code/game/objects/items/weapons/handcuffs.dm

```diff
@@ -229,3 +218,44 @@
 	throwforce = 0
 	breakouttime = 0
 	cuffsound = 'sound/weapons/cablecuff.ogg'
+
+//////////////////////////////
+// MARK: CRAFTING
+//////////////////////////////
+/obj/item/restraints/handcuffs/cable/attackby(obj/item/I, mob/user as mob, params)
+	..()
+	if(istype(I, /obj/item/stack/rods))
+		var/obj/item/stack/rods/R = I
+		if(!R.use(1))
+			to_chat(user, "<span class='warning'>You need one rod to make a wired rod!</span>")
```

Comment:
```suggestion
			to_chat(user, "<span class='warning'>[R.amount > 1 ? "These rods" : "This rod"] somehow can't be used for crafting!</span>")
```
If `use()` returns `FALSE`, it means that it wasn't able to be used, but you do need at least one rod to trigger this